### ISSUE TITLE
cython+gradient+array

### ISSUE THREAD

**Issue by rodiram** [scikit-hep/iminuit#268]
Dear developers, if you have 2 min to provide an example about how to pass an array of parameters in the gradient function with Cython, that would help me a lot. I did not find one in the sources, you mention it is possible but I was not able to figure out how to do it ..
Something like :
```
cpdef double[::1] gradient(self,double[::1] par):
```
instead of
```
cpdef double[::1] gradient(self, double p1,double p2,double p3)
```
Thank you very much !

**Comment by cdeil** [comment]
There's a few tests here passing a Python gradient function:
https://github.com/iminuit/iminuit/blob/master/iminuit/tests/test_iminuit.py

There is none for Cython and I don't know how to do it.

If anyone figures it out, please consider adding an example to the docs, e.g. as a new notebook showing one simple example using gradient from Python and Cython here:
http://iminuit.readthedocs.io/en/latest/tutorials.html

**Comment by HDembinski** [comment]
While this is not fixed, I am closing this. We have a tutorial now how to pass gradients computed with Jax to iminuit. In general, the future for fast computation in Python is not Cython, but in tools like numba and jax.
